devicemodel/core/main.c

```diff
@@ -1003,6 +1003,10 @@ main(int argc, char *argv[])
 		usage(1);
         }
 
+	if (lapic_pt == true && is_rtvm == false) {
+		lapic_pt = false;
+		pr_warn("Only realtime VM support local APIC pass through. '--lapic_pt' is invalid here.\n");
```

Comment:
How about the following text instead, would it be accurate?
```suggestion
		pr_warn("Only a Realtime VM can use local APIC pass through, '--lapic_pt' is invalid here.\n");
```

Reply:
updated